web/modules/custom/joinup_communities/eupl/eupl.behat.inc

```diff
@@ -77,6 +79,43 @@ class EuplSubContext extends DrupalSubContextBase {
     ])->save();
   }
 
+  /**
+   * Clicks the info icon next to the SPDX licence header.
+   *
+   * @param string $spdx
+   *   The SPDX licence ID.
+   *
+   * @throws \Exception
+   *   Thrown if the info icon could not be found.
+   *
+   * @Given I click the info icon of the :spdx licence( table cell)
+   */
+  public function clickLicenceInfoIcon(string $spdx): void {
+    $xpath = '//td[' . $this->xpathHasClassSelector('licence-comparer__header', FALSE) . ' and contains(., "' . $spdx . '")]//span[' . $this->xpathHasClassSelector('icon--info', FALSE) . ']';
```

Comment:
Yeah this is what I mean, let's just put the XPath expression here instead of complicating things by splitting it across different functions.

Reply:
Done.